src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1693,9 +1714,17 @@ void Partitioner::optimize(const std::string& func_name) {
 
             ov::parallel_for(func_group.refs.size(), [&](std::size_t f_idx) {
                 auto& funcall = func_group.refs[f_idx].get();
-                ov::Tensor cw = funcall._closure[w_idx - f._param_offset];
-                ov::Tensor cz = z_idx != -1 ? funcall._closure[z_idx - f._param_offset] : ov::Tensor{};
-                ov::Tensor cs = funcall._closure[s_idx - f._param_offset];
+                // FIXME: assuming no transformations were applied to the tensor - since we are utilizing the original
```

Comment:
@dmatveev is such cases need to be covered please let me know

Reply:
Yes, unpack must be a part of lazy tensor hierarchy

Reply:
Done, thanks